### PR TITLE
New Heimdall google addon features

### PR DESCRIPTION
**Pull request by adityapranav-b**
This PR contains addition of custom tag selection for the testcases and also contains a field for custom environment.

heimdall

### CODE REVIEW

**Comment by vaibsaraswat** [comment]
@adityapranav-b can you please add some description here.

File: heimdall.google-addon/execution.html [hunk 1]
```diff
@@ -16,72 +16,95 @@
...
+         document.getElementById('addbranch').style = "visibility: display; height: 50px;width: 300px;";
+      }
+      else {
+         document.getElementById('addbranch').style = "visibility: hidden; height: 0px;width: 0px;";
+      }
+   };
+
+
+   function fun() {
+      var env = (document.getElementById("customEnv").value == "") ? $("#environment").val() : document.getElementById("customEnv").value;
```
Comment (ntantri) [inline heimdall.google-addon/execution.html]: Multiple getElementById for same tag.
Can you please change this to:

```
var customEnv = document.getElementById("customEnv");
var env = (customEnv && customEnv.value === "") ? $("#environment").val() : customEnv.value;
```

File: heimdall.google-addon/execution.html [hunk 2]
```diff
@@ -16,72 +16,95 @@
...
+
+      for (var i = 0; i < 3; i++) {
+         if (checkedBoxes[i].checked == true) {
+            tags = tags + ",@" + checkedBoxes[i].name;
+         }
+      }
+      tags = tags.substring(1, tags.length);
+
+      if (checkedBoxes[3].checked == true) {
+         suites = $("#suitesMultiSelect").val();
```
Comment (ntantri) [inline heimdall.google-addon/execution.html]: `suites` is not used below. As I see `$("#suitesMultiSelect").val()` directly being accessed. 
As well  I see multiple ways of  accessing an element,  one via `JQuery $` and another by `document.getElementById`

@vaibsaraswat can we get this discussed across the team to use one norm?

**Review by ntantri — CHANGES_REQUESTED** [review]

File: heimdall.google-addon/execution.html (hunk 1 above)
Comment (adityapranav-b) [reply heimdall.google-addon/execution.html]: Done @tan31989

File: heimdall.google-addon/execution.html (hunk 2 above)
Comment (adityapranav-b) [reply heimdall.google-addon/execution.html]: Done

File: heimdall.google-addon/execution.html [hunk 3]
```diff
@@ -16,72 +16,95 @@
...
+      }
+      else {
+         document.getElementById('addbranch').style = "visibility: hidden; height: 0px;width: 0px;";
+      }
+   };
+
+
+   function fun() {
+      var customEnv = document.getElementById("customEnv");
+      var env = (customEnv && customEnv.value === "") ? $("#environment").val() : customEnv.value;
```
Comment (ntantri) [inline heimdall.google-addon/execution.html]: why are we using two type of id fetch? One via `document.getElementById` and another by `$("#environment")` . Can't we stick to one way of getting value by id?

**Review by ntantri — CHANGES_REQUESTED** [review]

File: heimdall.google-addon/execution.html (hunk 3 above)
Comment (adityapranav-b) [reply heimdall.google-addon/execution.html]: @tan31989 Changed everything to document.getElementById

**Review by ntantri — APPROVED** [review]

**Pull request merged**